sections/shared/modals/TxConfirmationModal/TxConfirmationModal.tsx

```diff
@@ -223,22 +220,20 @@ export const TxConfirmationModal: FC<TxConfirmationModalProps> = ({
 								components={[<NoTextTransform />]}
 							/>
 							&nbsp;
-							<StyledTooltip
-								placement="top"
+							<CustomStyledTooltip
+								preset="top"
```

Comment:
Make sure this tooltip stays within the borders of the modal. Also it should not disappear as soon as a user tries to move the mouse away from the (i) icon so that they can click the actual "Learn more" link :)

Reply:
Example:

![Screen Shot 2022-09-19 at 10 07 45](https://user-images.githubusercontent.com/548702/191025641-7ff82b51-5057-403d-bf24-62d65baf870d.png)

Reply:
It has been fixed also :)